src/views/partials/header.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en" class="govuk-template wg">
+<html lang="<%= i18n.language %>" class="govuk-template wg">
```

Comment:
👍

Reply:
Please can you make the corresponding change in the consumer side header if it needs it? ta